<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testsuite errors="0" failures="1" name="io.cucumber.core.plugin.JUnitFormatter" skipped="0" tests="1" time="41.015">
    <testcase classname="CRUD Reagent" name="create new reagent" time="40.352">
        <failure message="expected: &lt;Success&gt; but was: &lt;&gt;" type="org.opentest4j.AssertionFailedError"><![CDATA[Given user is navigated to website "http://localhost:5050"..................passed
Then user is navigated to "login" page......................................passed
When user enters username as "admin"........................................passed
And user enters password as "admin".........................................passed
And user clicks on "SignIn" button..........................................passed
Then user is navigated to "Dashboard" page..................................passed
When user clicks on menu "Reagent"..........................................passed
And user clicks on sub-menu "Library".......................................passed
Then user is navigated to "Reagent Library" page............................passed
When user clicks on "Create Reagent" button.................................passed
Then user is navigated to "Create Reagent" page.............................passed
When user type reagent "name" as "Reagent 5"................................passed
Then "Reagent 5" is typed on "name" textbox.................................passed
When user select "Antibiotic" on "type".....................................passed
Then "Antibiotic" is selected on "type".....................................passed
When user type reagent "note" as "Create Reagent 1 with Antibiotic".........passed
Then "Create Reagent 1 with Antibiotic" is typed on "note" textbox..........passed
When user clicks on "Save" button...........................................passed
Then new "reagent" item is created with notification "Success" and "Reagents has been created".failed
And There is no new "culture reagent" item created..........................skipped

StackTrace:
org.opentest4j.AssertionFailedError: expected: <Success> but was: <>
	at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at org.junit.jupiter.api.AssertionUtils.failNotEqual(AssertionUtils.java:62)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1124)
	at stepDefinition.Steps.new_item_is_created_with_notification_and(Steps.java:142)
	at âœ½.new "reagent" item is created with notification "Success" and "Reagents has been created"(file:///C:/Users/smith/OneDrive/Documents/INTERN/Cell/cellsuite/cellsuite/src/test/resources/features/reagent/reagent.feature:27)
]]></failure>
    </testcase>
</testsuite>
